milestone-2.md

```diff
@@ -0,0 +1,29 @@
+#### Milestone 2
+
+<b>Audience:</b> Crypto-Native Users.
+<b>Overview:</b> Your tech-savvy near account holder is invited to try Peerfolio. After a brief chat with the Peerfolio Agent, they are convinced that they will use Peerfolio as their first venture into agent optimized portfolios.
+
+
+#### Requirements and Acceptance Criteria
+- As a user (cryto native) I should be able to onboard into the fully designed, mobile-first Peerfolio app, deployed on a v0.peerfolio.app domain, where I'm able to choose my preferred wallet to connect my existing near account, setup my trading account and initiate the purchase of a recommended set of tokens via a coinbase onramp flow, and confirm the creation of the agent's suggested allowance goal, which the agent will attempt to execute pending the market conditions.
+
+#### Nice to haves:
+- [Support withdrawal of select tokens to their external wallets](https://github.com/beneviolabs/ft-allowance-agent/issues/76)
+- Adds [withdraw all / close your trading account](https://github.com/beneviolabs/ft-allowance-agent/issues/75)
+- [Create a flow diagram to vet the use of webAuthn and fully plan this architecture](https://github.com/beneviolabs/ft-allowance-agent/issues/91)
+
+
+#### Major enhancements:
+- Product Design for multiplayer mode
+- [Begin implementing the latest designs](https://github.com/beneviolabs/ft-allowance-agent/issues/90)
+- [Define the scope of UX for crypto-natives vs mainstream consumers](https://github.com/beneviolabs/ft-allowance-agent/issues/87)
+- [Include the fiat-to-crypto deposit flows in the latest designs](https://github.com/beneviolabs/ft-allowance-agent/issues/88)
+- [Auto execute conditional swap goals given market conditions](https://github.com/beneviolabs/ft-allowance-agent/issues/26)
```

Comment:
Right now this epic carries some of the work to refactor the user experience to match the intended design besides implementing the automatic swapping. I think it'd be clearer to break that out into a separate epic just so that QA and deliverables are clearer so I've done that here: https://github.com/beneviolabs/ft-allowance-agent/issues/98